features/visitor_can_see_specific_car_information.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Visitor can see specific car information
+    As a visitor
+    In order to rent a car
+    I want to see more information about specific car I have selected
+
+    Background:
+        Given The following cars exist
+            |brand|model|year|price|
+            |Volvo|V60  |2010|200  |
```

Comment:
alt+shift+f

Reply:
I think this one: Cucumber (Gherkin) Full Support

Reply:
Done